help_to_heat/frontdoor/mock_epc_api.py

```diff
@@ -17,43 +17,31 @@ def load_test_reponse(file_name):
 
 
 class MockEPCApi:
-    def __init__(self, token):
-        self.token = token
-
-    def get_address_and_rrn(self, building, postcode):
-        return load_test_reponse("sample_address_and_rrn_response.json")
+    def get_address_and_lmk(self, building, postcode):
+        return load_test_reponse("sample_address_and_lmk_response.json")
 
     def get_epc_details(self, rrn):
         return load_test_reponse("sample_epc_response.json")
 
-    def get_access_token(self):
-        return load_test_reponse("sample_token_response.json")
-
```

Comment:
old API used access tokens, but this has now been removed, so test data no longer needs this